runtime/compiler/jq.vim

```diff
@@ -0,0 +1,28 @@
+" Vim compiler file
+" Compiler:	jq
+" Maintainer:	Vito <vito.blog@gmail.com>
+" Last Change:	2024 Apr 17
+" Upstream: https://github.com/vito-c/jq.vim
+
+if exists('b:current_compiler')
+  finish
+endif
+let b:current_compiler = 'jq'
+
+let s:save_cpoptions = &cpoptions
+set cpoptions&vim
+
+if exists(':CompilerSet') != 2
```

Comment:
You can remove the `exists(':CompilerSet')` test. We just [removed](https://github.com/vim/vim/commit/408281e16a36c15eed10fbf0406fa8) this from the other compilers.